projects/milky-way/index.js

```diff
@@ -1,36 +1,27 @@
-const { queryContract } = require("../helper/chain/cosmos");
 const ADDRESSES = require('../helper/coreAssets.json')
+const sdk = require('@defillama/sdk')
+const { get } = require('../helper/http');
 
-const consts = {
-  MILKYWAY_CONTRACT: 'osmo1f5vfcph2dvfeqcqkhetwv75fda69z7e5c2dldm3kvgj23crkv6wqcn47a0',
-}
-
-/**
- *  {
- *       id: 1,
- *       batch_total_liquid_stake: '100000',
- *       expected_native_unstaked: '100000',
- *       received_native_unstaked: '100000',
- *       unstake_request_count: 1,
- *       next_batch_action_time: '0',
- *       status: 'received'
- *     }
- */
+const milkywayDelegationAddress = "celestia1vxzram63f7mvseufc83fs0gnt5383lvrle3qpt"
+const celestiaCoinGeckoId = "celestia";
 
 async function tvl() {
-  const { api } = arguments[3]
-  const data = await queryContract({ contract: consts.MILKYWAY_CONTRACT, chain: api.chain, data: { state: {} } });
-  const  {batches} = await queryContract({ contract: consts.MILKYWAY_CONTRACT, chain: api.chain, data: { batches: {} } });
-  const token = 'ibc/'+ADDRESSES.ibc.TIA
-  //  when calculating TVL, current unbonding batches with TIA should be added since they are 'locked' inside the contract at that current point in time
-  batches.filter(b => b.status !== 'received').forEach((b) => api.add(token, b.expected_native_unstaked))
```

Comment:
hi @honeybeara4 while testing your changes, noticed that it was approx 20% lower, I think it is because it missed the TIA that is in the process of being unstaked? like 26 in the old adapter

Reply:
That's right, it does not count unbonding TIA, since that's not in the "staking pool". 

Same as how tokens that are in the process of unbonding are not counted in TVL totals for Lido, Stride, etc. With this PR, those TVL comparisons become apples to apples.

Reply:
@honeybeara4 havent looked at lido & stride, but if they dont include it, then it is mistake in that adapter implementation. These tokens are still held by/coming to smart contract controlled by milky way team, so should count towards their tvl?